__tests__/functions/write-file.test.ts

```
@@ -0,0 +1,88 @@
+import fs from "fs";
+import { writeFile } from "../../src/functions/write-file";
+
+jest.mock("fs");
+jest.mock("../../src/config", () => ({
+	config: { format: "js", outputNameFormat: "kebab" }
+}));
+
+describe("It should throw an error", () => {
+	test("Without first parameter", () => {
+		expect(() => {
+			// @ts-ignore
+			writeFile(undefined, "lorem", "ipsum");
+		}).toThrow();
+	});
+
+	test("Without second parameter", () => {
```

Should fail if `path` is not provided.